components/backends/trtllm/README.md

```
@@ -202,6 +244,120 @@ DISAGGREGATION_STRATEGY="prefill_first" ./launch/disagg.sh
+        }
+    ],
+    "stream": false,
+    "max_tokens": 160
+}'
+```
+
+## Supported Multimodal Models
+
+Multimodel models listed [here](https://github.com/NVIDIA/TensorRT-LLM/blob/v1.0.0rc0/examples/pytorch/README.md) are supported by dynamo.
+
+## Using Pre-computed Embeddings (Experimental)
```

markdownlint-cli2 (0.17.2)

Multiple headings with the same content

(MD024, no-duplicate-heading)